pkg/wakaama/patches/0014-Use-lwm2m_strdup-instead-of-strdup.patch

```diff
@@ -0,0 +1,25 @@
+From 47d78e80ceb80a06e0cb9b0b7f7c959d04bd2c22 Mon Sep 17 00:00:00 2001
+From: Christian Manal <moenoel@informatik.uni-bremen.de>
+Date: Mon, 5 Feb 2018 15:52:10 +0100
+Subject: [PATCH] Use lwm2m_strdup() instead of strdup()
+
+---
+ examples/client/object_security.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/examples/client/object_security.c b/examples/client/object_security.c
+index bfb5f85..d91f6e5 100644
+--- a/examples/client/object_security.c
++++ b/examples/client/object_security.c
+@@ -540,7 +540,7 @@ lwm2m_object_t * get_security_object(int serverId,
+             targetP->securityMode = LWM2M_SECURITY_MODE_PRE_SHARED_KEY;
+             if (bsPskId)
+             {
+-                targetP->publicIdentity = strdup(bsPskId);
```

Comment:
@moenoel do you remember what was the reason for using `lwm2m_strdup` instead of `strdup`?

Reply:
One of the reasons was consistency, as Wakaama defines prototypes for a few functions that it expects the specific implementation to fill in (e.g. `lwm2m_malloc()` and `lwm2m_free()`) and it just seemed to make sense to use those wrappers throughout the Wakaama related code.

There was at least one other reason, but I can't for the life of me remember what that was specifically.